GOVERNANCE.md

```diff
@@ -0,0 +1,28 @@
+# auth-gateway Governance
+
+This document defines project governance for the project.
+
+## Voting
+
+todo
+
+## Changes in Maintainership
+
+Contributors who are interested in becoming a maintainer, if performing relevant responsibilities, should discuss their interest with the existing maintainers. New maintainers must be nominated by an existing maintainer and must be elected by 2/3 majority vote.
+
+We do not expect anyone to make a permanent commitment to be a auth-gateway maintainer forever. After all, circumstances change, people get new jobs, new interests, and may not be able to continue contributing to the project. At the same time, we need to keep the list of maintainers current in order to have effective governance. People may be removed from the current list of maintainers via one of the following ways:
+
+
+* They can resign
+* If they stop contributing to the project for a period of 6 months or more
+* By a 2/3 majority vote from active maintainers
```

Comment:
The same comment I wrote above applies here too. I wrote this as a placeholder. It is open to change